src/app/content/highlights/components/DisplayNote.tsx

```diff
@@ -53,6 +57,11 @@ const DisplayNote = React.forwardRef<HTMLElement, Props>((
     }
   }, [isFocused]);
 
+  React.useEffect(() => {
+    const refElement = confirmationRef.current ? confirmationRef : element;
+    onHeightChange(highlight.id, refElement);
+  }, [element, highlight, confirmationRef, confirmingDelete]);
```

Comment:
It's was here initially because `Confirmation` has position: absolute and I wanted to have direct access. 
I've moved logic with `onHeightChange` to `DisplayNote` and `EditCard`